.goreleaser.yml

```diff
@@ -56,6 +56,8 @@ release:
       name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
   # If you want to manually examine the release before its live, uncomment this line:
   # draft: true
+  prerelease: auto
+  make_latest: "{{ not .Prerelease }}"
```

Comment:
As per https://goreleaser.com/customization/templates/

```yaml
make_latest: "{{ not .Prerelease }}"
```

is wrong and

```yaml
"{{ .Prerelease }}"
```
will evaluate to `pre`
It is not a boolean, so my change will not work.

QQ: Can I match a template check for
```yaml
"{{ .Prerelease == 'pre' }}"
```

I am assuming templating works the same as they do in Helm charts.

Reply:
It looks like templates aren't supported in `make_latest` yet. 
There's a PR to support this: https://github.com/goreleaser/goreleaser/issues/5418

Reply:
I think it's merged and available as per v2.6.0
https://github.com/goreleaser/goreleaser/pull/5420

Reply:
`{{ eq (.Prerelease | len) 0 }}`

https://go.dev/play/p/4_PNM98L86r

Reply:
Reverting the change.